src/main/java/uk/ac/cam/cl/dtg/isaac/dos/IsaacEventPage.java

```diff
@@ -69,6 +69,8 @@ public class IsaacEventPage extends Content {
 
   private Boolean allowGroupReservations;
 
+  private Boolean privateEvent;
```

Comment:
Since this is a boolean, it would be a bit more readable to call it `isPrivateEvent`. The same goes for every instance in which such a name has been used.